### PR TITLE
Teams js router

### PR DESCRIPTION
**Pull request by Tanza3D**
bit janky but works quite well.

note: need to use `switchUrl(url)` instead of <a> with href for links... ugh

### CODE REVIEW

File: teams/js/routing.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+    if (cur_page_parent != null) {
+        data.append('page', cur_page_parent.name);
+        data.append('subpage', cur_page.name);
+    } else {
+        data.append('page', cur_page.name);
+    }
+    xhr.onload = function () {
+        // do something to response
+        teams_page.innerHTML = xhr.responseText;
```
Comment (Coppertine) [inline teams/js/routing.js:78]: looks very dangerous to do... I hope we know what we're doing.. someone could intercept a php request...

**Review by Coppertine — APPROVED** [review]
I guess it's fine? trying to do a single page view is going to be scuff...

**Review by Coppertine — APPROVED** [review]
I guess it's fine? trying to do a single page view is going to be scuff...

File: teams/js/routing.js (hunk 1 above)
Comment (Tanza3D) [reply teams/js/routing.js:78]: should be fine

**Pull request merged**